pilot/pkg/model/config_test.go

```diff
@@ -707,3 +680,124 @@ func addRbacConfigToStore(configType, name, namespace string, store model.IstioC
 		t.Error(err)
 	}
 }
+
+type fakeStore struct {
+	model.ConfigStore
+	cfg map[string][]model.Config
+	err error
+}
+
+func (l *fakeStore) List(typ, namespace string) ([]model.Config, error) {
+	ret := l.cfg[typ]
+	return ret, l.err
+}
+
+func TestIstioConfigStore_QuotaSpecByDestination(t *testing.T) {
+	ns := "ns1"
+	l := &fakeStore{
+		cfg: map[string][]model.Config{
+			model.QuotaSpecBinding.Type: {
+				{
+					ConfigMeta: model.ConfigMeta{
+						Namespace: ns,
+						Domain:    "cluster.local",
+					},
+					Spec: &mccpb.QuotaSpecBinding{
+						Services: []*mccpb.IstioService{
+							{
+								Name:      "a",
+								Namespace: ns,
+							},
+						},
+						QuotaSpecs: []*mccpb.QuotaSpecBinding_QuotaSpecReference{
+							{
+								Name: "request-count",
+							},
+							{
+								Name: "does-not-exist",
```

Comment:
would be good to test this, i guess we only see a warning right?

Reply:
yes, we only see warning. I saw it in code coverage. It will take a bit more work to verify this.